langchain/tools/multi_line_human/tool.py

```diff
@@ -0,0 +1,47 @@
+"""Tool for asking human input."""
+
+from typing import Callable
+
+from pydantic import Field
+
+from langchain.tools.base import BaseTool
+
+
+def _print_func(text: str) -> None:
+    print("\n")
+    print(text)
+
+def input_func():
+    print("Insert your text. Press Ctrl-D (or Ctrl-Z on Windows) to end.")
+    contents = []
+    while True:
+        try:
+            line = input()
+        except EOFError:
+            break
+        contents.append(line)
+    return "\n".join(contents)
+
+class MultiLineHumanInputRun(BaseTool):
+    """Tool that adds the capability to ask user for multi line input."""
+
+    name = "MultiLineHuman"
+    description = (
+        "You can ask a human for guidance when you think you "
+        "got stuck or you are not sure what to do next. "
+        "The input should be a question for the human."
```

Comment:
space at end

Reply:
Hey, thanks for the review, I've just updated it.